src/main/java/mate/academy/MergeSortAction.java

```diff
@@ -1,8 +1,54 @@
 package mate.academy;
 
-public class MergeSortAction {
+import java.util.Arrays;
+import java.util.concurrent.RecursiveAction;
+
+public class MergeSortAction extends RecursiveAction {
+    private static final int THRESHOLD = 1;
```

Comment:
The THRESHOLD constant is set to 1, which means that the sorting task will be split until it reaches a single element, which is not efficient for parallel execution. Consider increasing the threshold to a value that makes sense for parallel execution, such as 10 or 30, to reduce the overhead of task management.

Reply:
If I make it bigger than '1' two tests will fail because we cannot guarantee the same result